README.md

```diff
@@ -507,6 +509,526 @@ source scripts/makeAllPlots.sh figures/ResponseUniformityMaps
+			[BEGIN_DATA]
+			VAR_INDEP,VAR_DEP,VAR_INDEP_ERR,VAR_DEP_ERR
+			1.0,20.38,0.0,0.0
+			62.0,20.26,0.0,0.0
+			123.0,20.13,0.0,0.0
+			184.0,20.14,0.0,0.0
+			245.0,20.11,0.0,0.0
+			306.0,20.11,0.0,0.0
+			367.0,20.04,0.0,0.0
+			428.0,20.05,0.0,0.0
+			489.0,20.03,0.0,0.0
+			550.0,20.07,0.0,0.0
+			611.0,20.05,0.0,0.0
+			672.0,20.07,0.0,0.0
+			733.0,20.01,0.0,0.0
+			794.0,20.07,0.0,0.0
+			855.0,19.98,0.0,0.0
+			916.0,19.98,0.0,0.0
+			...................
+			...................
+			...................
+			3112.0,19.39,0.0,0.0
+			3173.0,19.32,0.0,0.0
+			3234.0,19.32,0.0,0.0
+			3295.0,19.35,0.0,0.0
+			3356.0,19.23,0.0,0.0
+			3417.0,19.25,0.0,0.0
+			3478.0,19.25,0.0,0.0
+			3539.0,19.29,0.0,0.0
+			3600.0,19.2,0.0,0.0
+			[END_DATA]
+			[BEGIN_FIT]
+			Fit_Color = 'kRed';
+			Fit_Formula = '[0]*TMath::Exp([1]*x)';
+			Fit_LegEntry = 'Fit_GE11-X-L-CERN-0002';
+			Fit_Line_Size = '1';
+			Fit_Line_Style = '1';
+			Fit_Name = 'Fit_GE11-X-L-CERN-0002';
+			Fit_Option = 'R';
+			Fit_Param_IGuess = 'AMPLITUDE,6.2e-05';
+			Fit_Perform = 'true';
+			Fit_Range = '0,3600';
+			[END_FIT]
+		[END_PLOT]
+	[END_CANVAS] 
+	
+In the case that more than one input files are selected the output name is automatically set to be: **config\_QC3\_LS2\_Pres\_vs\_Time\_AllDet.cfg**
+
+**QC4_HV.py :**
+
+	--CanvTitleX=Divider Current #left(#muA#right)
+	--CanvTitleY=Applied Voltage #left(kV#right)
+	--SelectColumnX=3
+	--SelectColumnY=1
+	--SelectRowStart=3
+	--SelectRowEnd=37
+	--CanvRangeX=0,1000
+	--CanvRangeY=0,7
+	--YaxisScale
+	--LatexLines=0.62,0.86, #splitline{LS2}{Detector~Production}
+	--LatexLines=0.62,0.27, Gas~=~CO_{2}
+
+	Moreover if the Fit option is used (example: python python/QC4_HV.py —-file=QC4_GE11-X-L-CERN-0001_20170601.xlsm —-Fit)
+	the following options are set for the fit:
+
+	--Fit
+	--FitFormula=[0]*x+[1]
+	--FitParamIGuess=Req,5   (where Req is the measured resistance from the excel file) 
+	--FitRange=0,1000
+	
+Example of use: As in steps 1 and 2 of QC3.py script execute:
+
+	1. source scripts/setup_CMS_GEM.sh
+	2. python2.7 python/QC4_HV.py --file=Filename.xlsx (--Fit)
+	
+The output filename in case of one input file will be: 
+	
+* config\_V\_vs\_Imon\_Filename.cfg
+
+or 
+
+* config\_QC4\_LS2\_V\_vs\_Imon\_AllDet.cfg
+
+in case of multiple input files (	python2.7 python/QC4\_HV.py --file=Filename.xlsx --file=Filename2.xlsx ..)
+	
+
+**QC4_SS.py :**
+
+	--CanvTitleX=Divider Current #left(#muA#right)
+	--CanvTitleY=Spurious Signal R_{SS} #left(Hz#right)
+	--LatexLines=0.62,0.86, #splitline{LS2}{Detector~Production}
+	--LatexLines=0.62,0.27, Gas~=~CO_{2}
+	--LatexLines=0.58,0.73, Readout~=~GEM3B
+	--SelectColumnX=3
+	--SelectColumnY=7
+	--SelectRowStart=3
+	--SelectRowEnd=37
+	--CanvRangeX=0,1000
+	--CanvRangeY=0,50
+	--SetErrY
+	--SelectColumnErrY=8
+	
+Example of use: As in steps 1 and 2 of QC3.py script execute:
+
+	1. source scripts/setup_CMS_GEM.sh
+	2. python2.7 python/QC4_SS.py --file=Filename.xlsx (--Fit)
```

Comment:
Don't put this as an itemized list, it doesn't format then as a block of code to be executed.  e.g. always use:

```
command1
command2
command3
```

Reply:
done